lms/templates/student_account/login_and_register.html

```diff
@@ -1,6 +1,7 @@
 <%!
```

Comment:
You will need to add a page directive which escapes all strings by default to the top of this template in order to resolve the [safe template linter](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/safe_templates.html#safe-template-linter) quality [errors](https://build.testeng.edx.org/job/edx-platform-quality-pr/34621/artifact/edx-platform/reports/safelint/safelint.report) you are getting on this PR.
```
<%page expression_filter="h"/>
```
For more details, see [this](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/safe_templates.html#set-html-escaping-filter-as-default).